opensearch-operator/controllers/configurationController.go

```diff
@@ -0,0 +1,70 @@
+package controllers
```

Comment:
Got the separate to configController, looks good, is ti working for sure ?

Reply:
Verified it by running the operator locally when testing out the security controller stuff